src/stories/base/Modal.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import type { ReactElement } from 'react';
+import { useState } from 'react';
+import Modal from '@base/Modal';
+import type { ModalProps } from '@base/Modal/types';
+
+export default {
+  title: 'Component/Modal',
```

Comment:
```suggestion
  title: 'Component/base/Modal',
```
로 바꾸면 스토리북에서 좀 더 일관되게 적용될 수 있을 듯 합니다.